packages/composites/outlier-detection/src/source.ts

```diff
@@ -1,39 +1,9 @@
-import { Execute } from '@chainlink/types'
-import * as GenesisVolatility from '@chainlink/genesis-volatility-adapter'
-import * as XBTO from '@chainlink/xbto-adapter'
-import * as dxFeed from '@chainlink/dxfeed-adapter'
-import { util } from '@chainlink/ea-bootstrap'
+import { AdapterImplementation } from '@chainlink/types'
+// source adapters
+import GenesisVolatility from '@chainlink/genesis-volatility-adapter'
+import XBTO from '@chainlink/xbto-adapter'
+import dxFeed from '@chainlink/dxfeed-adapter'
 
-export enum SourceDataProvider {
-  XBTO = 'xbto',
-  GenesisVolatility = 'genesisvolatility',
-  dxFeed = 'dxfeed',
-}
+export const adapters: AdapterImplementation[] = [GenesisVolatility, XBTO, dxFeed]
```

Comment:
Part of my comment above. We should just remove any references of a specific set of adapters and instead allow any adapters to be used.